src/store/messagesStore.js

```diff
@@ -286,6 +288,64 @@ const actions = {
 		return response.status
 	},
 
+	/**
+	 * Creates a temporary message ready to be posted, based
+	 * on the message to be replied and the current actor
+	 *
+	 * @param {object} context default store context;
+	 * @param {string} text message string;
+	 * @param {string} token conversation token;
+	 * @param {string} uploadId upload id;
+	 * @param {int} index index;
+	 * @param {object} file file to upload;
+	 * @param {string} localUrl local URL of file to upload;
+	 * @returns {object} temporary message
+	 */
+	createTemporaryMessage(context, { text, token, uploadId, index, file, localUrl }) {
+		const messageToBeReplied = context.getters.getMessageToBeReplied(token)
+		const date = new Date()
+		let tempId = 'temp-' + date.getTime()
+		const messageParameters = {}
+		if (file) {
+			tempId += '-' + uploadId + '-' + Math.random()
+			messageParameters.file = {
+				'type': 'file',
+				'file': file,
```

Comment:
```suggestion
				file,
```

Reply:
this is just copy pasted from the old copy, but probably a good idea to adjust. I'll adjust them all